### DISCUSSION TITLE
Supported Currencies

### DISCUSSION THREAD

**Discussion by danpastori** [serversideup/financial-freedom#111] [Project Discussions]
## 🎯 GOAL: Make Financial Freedom Work Anywhere
We want to make Financial Freedom work with any currency. Right now, we need to take a few of the bigger ones, but structure the system in a way to easily add other currency support. 

Related to #109 , I have seeded the options with:

- USD
- EUR
- GBP

## 🙋‍♂️ What other currencies should we start with supporting?
What other currencies should be supported from the start? And how many decimal places? I found this resource which could be helpful: https://docs.adyen.com/development-resources/currency-codes/

**Comment by jaydrogers** [comment]
One thing that came to mind is some currencies might also have different separators.

For example, `23,49 € EUR`

**Reply by danpastori** [reply]
Definitely! I'm thinking of using either DineroJS: https://github.com/serversideup/financial-freedom/discussions/73
or
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Most likely the IntlNumberFormat since it's built into JS itself and should have full browser support

**Comment by RokSiEu** [comment]
I would add two more, just for potential higher user reach via content on website  

- https://en.wikipedia.org/wiki/Japanese_yen 
- https://en.wikipedia.org/wiki/Indian_rupee

**Reply by danpastori** [reply]
Sweet. I added them as options! I'll ensure the database supports these formats moving forward. Luckily the most decimal places is 3 for any currency it looks like
